### PR TITLE
y-axis real interval adjusted to include all plot objects

### PR DESCRIPTION
**Pull request by ashalper-usgs**

### CODE REVIEW

**Comment by mhines-usgs** [comment]
pulling this locally to test out now.

File: inst/extdata/correctionsataglance/correctionsataglance-example.json [hunk 1]
```diff
@@ -8,9 +8,9 @@
       "description": "NWISWeb reference line displayed to WSC level users",
       "periods": [
         {
-          "startTime": "2016-03-28T00:00:00-06:00",
+          "startTime": "0001-01-01T00:00:00Z",
```
Comment (mhines-usgs) [inline inst/extdata/correctionsataglance/correctionsataglance-example.json:11]: Weird...

**Comment by mhines-usgs** [comment]
This looks good to me @ashalper-usgs !

File: R/vdiagram-render.R [hunk 2]
```diff
@@ -51,17 +51,32 @@ createVdiagram <- function(data) {
...
-  vplot <- gsplot(mar=c(7, 3, 4, 2), yaxs = "r", xaxs = "r") %>%
-    points(NA,NA, ylab=styles$plot$ylab, xlab=styles$plot$xlab)
+  vplot <- gsplot(mar = c(7, 3, 4, 2), yaxs = "r", xaxs = "r") %>%
+    points(NA, NA, ylab = styles$plot$ylab, xlab = styles$plot$xlab)
   
   vplot <- do.call(grid, append(list(object=vplot), styles$grid))
   vplot <- do.call(axis, append(list(object=vplot), styles$axis))
-  vplot <- do.call(abline, append(list(object=vplot, a=vdiagramData$maxStage), styles$maxStageLine))
-  vplot <- do.call(abline, append(list(object=vplot, a=vdiagramData$minStage), styles$minStageLine))
+  vplot <-
```
Comment (mhines-usgs) [inline R/vdiagram-render.R:59]: any reason you expanded these two calls? not any code changes as far as I can tell

Comment (ashalper-usgs) [reply R/vdiagram-render.R:59]: Auto-formatted with RStudio, didn't like how that looked, so I cleaned it up a bit. Don't mind much if it's set back the way it was.

Comment (mhines-usgs) [reply R/vdiagram-render.R:59]: ah, no no worries. I don't mind it this way, just took me a minute to realize it's not different!

**Pull request merged**